.gitpod.dockerfile

```diff
@@ -2,6 +2,15 @@ FROM gitpod/workspace-postgres
 
 # Install Ruby
 ENV RUBY_VERSION=3.0.2
+
+# Required for Gitpod to persist the gems in the container
+# See https://community.gitpod.io/t/persisting-home-gitpod/1296/2 and
+# https://github.com/gitpod-io/gitpod/issues/905#issuecomment-578747772
+ENV GEM_HOME /workspace/.rvm
```

Comment:
After reading a bit more, the issue appeared to be I was also missing the `GEM_PATH` and `BUNDLE_PATH`. I'll let the more knowledgeable Ruby folks chime in here. (See https://stackoverflow.com/a/42677414/77814)

Reply:
Now that I'm using `gp env` and overriding because of overriding rvm's `gemdir`, the environment variables being set here are probably not necessary. I'll check tomorrow.

Reply:
The linked thread concludes by stating that none of this works properly, including `gp env`. The author then points to their own PR which updates the `Dockerfile` for the workspace, so none of this should be necessary. Check if `~/.rvmrc` contains an entry for `rvm_gems_path=/workspace/.rvm`.

Reply:
Great call @citizen428! I found this in the file.

```
rvm_gems_path=/home/gitpod/.rvm
```

I'll go ahead and update that path and then I think we should be golden and I can remove the other things I tried.